src/tests/cli/firewall-cmd.at

```diff
@@ -4,7 +4,12 @@ m4_ifdef([TESTING_FIREWALL_OFFLINE_CMD], [], [
 
 FWD_START_TEST([basic options])
     AT_KEYWORDS(panic reload)
```

Comment:
Please add keyword `gh808`. Then you can run this individual test case with `make check TESTSUITEFLAGS="-k gh808"`.